docker-compose.yml

```diff
@@ -53,7 +49,6 @@ services:
 
```

Comment:
The docker hosts `wiremock` and `community-api` should still work here given the name of the service still matches. Removing the `container_name` should not break it and we've got the apps running locally to test.